stream-chat-android-ui-common/src/main/kotlin/com/getstream/sdk/chat/adapter/MessageListItem.kt

```diff
@@ -41,6 +41,7 @@ public sealed class MessageListItem {
         val isMine: Boolean = false,
         val messageReadBy: List<ChannelUserRead> = listOf(),
         val isThreadMode: Boolean = false,
+        val isMessageRead: Boolean = true,
```

Comment:
Defaults to "true" as the most common case is when channel is read almost to the end

Reply:
it's a change in public API. Should be stated in CHANGELOG